src/message/OutboxMessageContainer.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { connectActionSheet } from '@expo/react-native-action-sheet';
+
+import type { Actions, Auth } from '../types';
+import htmlToDomTree from '../html/htmlToDomTree';
+import renderHtmlChildren from '../html/renderHtmlChildren';
+import MessageFull from './MessageFull';
+import MessageBrief from './MessageBrief';
+import { getAuth } from '../selectors';
+import boundActions from '../boundActions';
+
+class OutboxMessageContainer extends PureComponent {
+  props: {
+    actions: Actions,
+    message: Object,
+    auth: Auth,
+    twentyFourHourTime: boolean,
+    isBrief: boolean,
+  };
+
+  static defaultProps = {
+    twentyFourHourTime: false,
+  };
+
+  render() {
+    const { message, auth, actions, twentyFourHourTime, isBrief } = this.props;
```

Comment:
Initially I did that but it caused a lot of if condition's in the component, hence I decided to make it's own component!